js/packages/teams-ai/src/authentication/Authentication.spec.ts

```diff
@@ -0,0 +1,201 @@
+// import * as sinon from 'sinon';
+// import { BotAuthentication } from './BotAuthentication';
+// import { MessagingExtensionAuthentication } from './MessagingExtensionAuthentication';
+// import { Application } from '../Application';
+// import { OAuthPromptSettings } from 'botbuilder-dialogs';
```

Comment:
Why is this file commented out?

Reply:
i'm still implementing the test files, you can ignore this for now. The next couple commits will replace this with unit tests.